server/game/cards/13.3-PoS/GuardDuty.js

```diff
@@ -0,0 +1,22 @@
+const DrawCard = require('../../drawcard.js');
+
+class GuardDuty extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction(
+            { faction: 'thenightswatch' }
+        );
+        this.reaction({
+            when: {
+                onDeclaredAsDefender: event => event.card === this.parent
```

Comment:
Missing check that 1. the parent can stand, 2. the parent is currently knelt (which will probably need a check that there is a parent)